front/app/containers/Admin/projects/project/timeline/edit.tsx

```diff
@@ -175,172 +128,129 @@ class AdminProjectTimelineEdit extends PureComponent<
           })
         )
         .subscribe((phaseFiles) => {
-          this.setState({ inStatePhaseFiles: phaseFiles as FileType[] });
+          setInStatePhaseFiles(phaseFiles as FileType[]);
```

Comment:
This will be removed in the next step, correct?

Reply:
Correct :)